<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ColorMappingChooser</class>
 <widget class="QDockWidget" name="ColorMappingChooser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>305</width>
    <height>538</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Color Mapping</string>
  </property>
  <widget class="QWidget" name="content">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="relatedRenderView">
      <property name="text">
       <string>--title--</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="scalarsGroupBox">
      <property name="title">
       <string>&amp;Scalars</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <layout class="QFormLayout" name="formLayout">
       <item row="1" column="0">
        <widget class="QLabel" name="componentLabel">
         <property name="text">
          <string>&amp;Component</string>
         </property>
         <property name="buddy">
          <cstring>componentSpinBox</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="2">
        <widget class="QComboBox" name="scalarsComboBox"/>
       </item>
       <item row="1" column="1">
        <widget class="QSpinBox" name="componentSpinBox"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="maxLabel">
         <property name="text">
          <string>&amp;Max</string>
         </property>
         <property name="buddy">
          <cstring>maxValueSpinBox</cstring>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="DoubleSpinBox" name="maxValueSpinBox"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="minLabel">
         <property name="text">
          <string>M&amp;in</string>
         </property>
         <property name="buddy">
          <cstring>minValueSpinBox</cstring>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="DoubleSpinBox" name="minValueSpinBox"/>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="gradientGroupBox">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="title">
       <string>&amp;Gradient</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QComboBox" name="gradientComboBox"/>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="nanColorButton">
         <property name="text">
          <string>&amp;NaN-Color</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="legendGroupBox">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="title">
       <string>Color &amp;Legend</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QScrollArea" name="scrollArea">
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>267</width>
            <height>251</height>
           </rect>
          </property>
          <layout class="QFormLayout" name="formLayout_3">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item row="0" column="0">
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>&amp;Title</string>
             </property>
             <property name="buddy">
              <cstring>legendTitleEdit</cstring>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="legendTitleEdit">
             <property name="clearButtonEnabled">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>&amp;Position</string>
             </property>
             <property name="buddy">
              <cstring>legendPositionComboBox</cstring>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QComboBox" name="legendPositionComboBox">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
           <item row="2" column="0" colspan="2">
            <widget class="QCheckBox" name="legendAlignTitleCheckBox">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>&amp;Align title with color bar</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0" colspan="2">
            <widget class="QCheckBox" name="legendTransparentBackground">
             <property name="text">
              <string>&amp;Transparent Background</string>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>Asp&amp;ect Ratio</string>
             </property>
             <property name="buddy">
              <cstring>legendAspectRatio</cstring>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QDoubleSpinBox" name="legendAspectRatio">
             <property name="minimum">
              <double>5.000000000000000</double>
             </property>
             <property name="maximum">
              <double>100.000000000000000</double>
             </property>
            </widget>
           </item>
           <item row="5" column="0">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Title &amp;Font Size</string>
             </property>
             <property name="buddy">
              <cstring>legendTitleFontSize</cstring>
             </property>
            </widget>
           </item>
           <item row="5" column="1">
            <widget class="QSpinBox" name="legendTitleFontSize"/>
           </item>
           <item row="6" column="0">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>&amp;Label Font Size</string>
             </property>
             <property name="buddy">
              <cstring>legendLabelFontSize</cstring>
             </property>
            </widget>
           </item>
           <item row="6" column="1">
            <widget class="QSpinBox" name="legendLabelFontSize"/>
           </item>
           <item row="7" column="0">
            <widget class="QCheckBox" name="legendTickLabelsCheckBox">
             <property name="text">
              <string>&amp;Labels</string>
             </property>
            </widget>
           </item>
           <item row="7" column="1">
            <widget class="QCheckBox" name="legendRangeLabelsCheckBox">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>&amp;Range Labels</string>
             </property>
            </widget>
           </item>
           <item row="8" column="0">
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>N&amp;umber of Labels</string>
             </property>
             <property name="buddy">
              <cstring>legendNumLabelsSpinBox</cstring>
             </property>
            </widget>
           </item>
           <item row="8" column="1">
            <widget class="QSpinBox" name="legendNumLabelsSpinBox">
             <property name="maximum">
              <number>64</number>
             </property>
            </widget>
           </item>
           <item row="9" column="0">
            <widget class="QCheckBox" name="legendTickMarksCheckBox">
             <property name="text">
              <string>T&amp;ick Marks</string>
             </property>
            </widget>
           </item>
           <item row="9" column="1">
            <widget class="QCheckBox" name="legendSubTickMarksCheckBox">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>&amp;Sub-Tick Marks</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DoubleSpinBox</class>
   <extends>QLineEdit</extends>
   <header>gui/widgets/DoubleSpinBox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>legendTickMarksCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>legendSubTickMarksCheckBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>87</x>
     <y>555</y>
    </hint>
    <hint type="destinationlabel">
     <x>225</x>
     <y>555</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>scalarsGroupBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>gradientGroupBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>56</x>
     <y>65</y>
    </hint>
    <hint type="destinationlabel">
     <x>36</x>
     <y>217</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>scalarsGroupBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>legendGroupBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>98</x>
     <y>69</y>
    </hint>
    <hint type="destinationlabel">
     <x>110</x>
     <y>305</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>legendTickLabelsCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>legendRangeLabelsCheckBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>45</x>
     <y>449</y>
    </hint>
    <hint type="destinationlabel">
     <x>230</x>
     <y>449</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
